src/applications/personalization/profile/components/NameTag.jsx

```diff
@@ -8,12 +8,12 @@ import prefixUtilityClasses from 'platform/utilities/prefix-utility-classes';
 import { SERVICE_BADGE_IMAGE_PATHS } from '../constants';
 import { getServiceBranchDisplayName } from '../helpers';
 
-const ProfileHeader = ({
+const NameTag = ({
```

Comment:
I think we should also update line 119 `data-testid="profile-header"` => `data-testid="name-tag" `, which of course means updating some tests that use that test id.

Reply:
I agree, updated!